package.json

```diff
@@ -93,7 +94,8 @@
     "test-android-instrumentation": "yarn run docker-build-android && yarn run test-android-run-instrumentation",
     "test-android-unit": "yarn run docker-build-android && yarn run test-android-run-unit",
     "test-android-e2e": "yarn run docker-build-android && yarn run test-android-run-e2e",
-    "test-ios": "./scripts/objc-test.sh test"
+    "test-ios": "./scripts/objc-test.sh test",
+    "test-typescript": "dtslint types"
```

Comment:
I don't know if you explored this, but there is https://github.com/jest-community/jest-runner-tsd allowing you to run type tests with Jest instead of adding a new tool.

You might have ported tests from DT, in which case using the same tool is easier, I guess 😀 

---

Also, super excited you've done this! 🎉